Loop/Views/SettingsViewModel.swift

```diff
@@ -51,7 +51,8 @@ public class SettingsViewModel: ObservableObject {
     var cgmManagerSettingsViewModel: DeviceViewModel
     var therapySettings: TherapySettings
     let supportedBasalRates: [Double]?
-    
+    let supportedBolusVolumes: [Double]?
```

Comment:
Same note here as in my comment on https://github.com/tidepool-org/LoopKit/pull/147 re combining the optional properties since they're tied together